doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -105,6 +105,11 @@ Partition Manager
 
 There are no entries for this section yet.
 
+Secure Partition Manager
+========================
+
+All EGU peripherals, instead of just EGU1 and EGU2, are now configurable to be non-secure and are configured as non-secure by default.
```

Comment:
```suggestion
* All EGU peripherals, instead of just EGU1 and EGU2, are now configurable to be non-secure and are configured as non-secure by default.
```